app/src/main/res/values/strings.xml

```diff
@@ -328,13 +328,13 @@
     <string name="theme_solarized">Solarized</string>
     <string name="theme_solarized_dark">Solarized Dark</string>
 
-    <string name="switch_to_remote">Switch to remote after media start</string>
-    <string name="keep_remote_above_lockscreen">Keep remote above lockscreen</string>
-    <string name="pref_keep_screen_on">Keep screen on when using remote</string>
-    <string name="show_notification">Show notification while playing</string>
-    <string name="pause_during_calls">Pause playing while phone in a call</string>
-    <string name="use_hardware_volume_keys">Use volume keys to control volume</string>
-    <string name="vibrate_on_remote">Vibrate on remote button press</string>
+    <string name="switch_to_remote">Show on media start</string>
+    <string name="keep_remote_above_lockscreen">Show over lockscreen</string>
+    <string name="pref_keep_screen_on">Stay awake</string>
+    <string name="show_notification">Notifications</string>
```

Comment:
I prefer the previous version of this one. 
Note that all preferences now start with a Verb, except this one (or ones that lead to dialogs), and it would be nice if this rule applied to all of them.